Features/Inventory/PlayerInventory/UI_InventoryItem/InventoryItemView.cs

```diff
@@ -0,0 +1,98 @@
+using Godot;
+using untitledplantgame.Common;
+
+namespace untitledplantgame.Inventory.PlayerInventory.UI_InventoryItem;
+
+public partial class InventoryItemView : Control
+{
+	[Export] private Texture2D _specificItemIcon;
+	[Export] private Label _displayItemName;
+	[Export] private Label _itemCurrentQuantity;
+	[Export] private TextureRect _itemTextureRect;
+
+	public int Id;
+	private Logger _logger;
+	public ItemStack ItemStack;
```

Comment:
might wanna sort the variables manually by C# coding guideline for godot while we still didn't figure out csharpier options for that?